tests-ng/plugins/kernel_configs.py

```diff
@@ -0,0 +1,54 @@
+import glob
+import platform
+
+import pytest
+
```

Comment:
other plugins are typically implemented as a class, is there a good reason to do it differently here? not exactly sure if that class pattern is really the best, but I would argue that consistency is important here, unless there is a really good reason to do it differently

Reply:
I don't really see a reason to overengineer this into a class to be completely honest. But I can if you'd prefer.